install/scripts/docker_setup_forseti.sh

```diff
@@ -38,11 +38,11 @@ fi
 # This assumes the script is run from the top of the source-tree.
 if [ -x "$(command -v docker)" ]; then
     echo "Building our Docker base image... "
-    docker build -t forseti/base -f install/docker/base --no-cache .
-    echo "Building our Forseti image from the Docker base image... "
-    docker build -t forseti/build -f install/docker/forseti --no-cache .
+    docker build --target build -t forseti/build .
+    # echo "Building our Forseti image from the Docker base image... "
```

Comment:
Why comment this echo output here?

Reply:
Thanks, the commented lines will be removed.